dogs.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>html200-adopt-a-dog</title>
+    <link rel="stylesheet" href="./styles/styles.css">
+  </head>
+  <body>
+
+
+    <header class="header-img">
+      <div class="b2box">
+        <img class="logohdr" src="./images/adoptadog-logo.svg" alt="Company Logo" id="adoptadog-logo">
+        <h2 class="hdrtxt">Seattle's premiere dog adoption agency</h2>
+      </div>
+        <a class="ckout" href="./checkout.html">Checkout</a>
+
+    </header>
+
+    <nav>
+      <ul>
+        <li>
+          <a href="./index.html">Home</a>
+        </li>
+        <li>
+          <a href="#">Dogs</a>
+        </li>
+        <li>
+          <a href="./blog.html">Blog</a>
+        </li>
+      </ul>
+    </nav>
+
+    <main class="wrapper">
+      <section class="flex-container">
+        <h2 class="div1">Here are all of our Dogs!</h2>
+          <article class="div2">
+
+            <img class="dogimg" src="./images/murphy-card.jpg" alt="Brown and White dog with black bakcground lovingly looking at camera">
+

```

Comment:
Would also suggest seeing if you can add some background/border to create the tile/card look!